src/test/java/org/sonarsource/sonarlint/ls/connected/TaintVulnerabilitiesCacheTests.java

```diff
@@ -63,39 +64,13 @@ void testIssueConversion() {
     assertThat(diagnostic.getData()).isEqualTo("issueKey");
   }
 
-  @Test
-  void testCacheOnlyUnresolvedTaintVulnerabilities() throws Exception {
-    var uri = new URI("/");
-    var taint = mock(ServerIssue.class);
-    when(taint.key()).thenReturn("key1");
-    when(taint.ruleKey()).thenReturn(SAMPLE_SECURITY_RULE_KEY);
-    when(taint.resolution()).thenReturn("");
-    when(taint.severity()).thenReturn("BLOCKER");
-    when(taint.getMessage()).thenReturn("Boo");
-
-    var resolvedTaint = mock(ServerIssue.class);
-    when(resolvedTaint.ruleKey()).thenReturn(SAMPLE_SECURITY_RULE_KEY);
-    when(resolvedTaint.resolution()).thenReturn("FIXED");
-
-    var notTaint = mock(ServerIssue.class);
-    when(notTaint.ruleKey()).thenReturn("java:S123");
-    when(notTaint.resolution()).thenReturn("");
-
-    underTest.reload(uri, List.of(taint, resolvedTaint, notTaint));
-
-    assertThat(underTest.getAsDiagnostics(uri)).hasSize(1);
```

Comment:
We don't check what this `getAsDiagnostics` method returns anymore, so it might be worth to keep a test on it ?